src/code/z_play.c

```diff
@@ -467,19 +477,21 @@ void Gameplay_Update(GlobalContext* globalCtx) {
 
         if (globalCtx->transitionMode) {
             switch (globalCtx->transitionMode) {
-                case 1:
-                    if (globalCtx->sceneLoadFlag != -0x14) {
-                        s16 sp6E = 0;
+                case TRANS_MODE_SETUP:
+                    if (globalCtx->transitionTrigger != TRANS_TRIGGER_OUT) {
+                        s16 csOffset = 0;
+                        
                         Interface_ChangeAlpha(1);
 
                         if (gSaveContext.cutsceneIndex >= 0xFFF0) {
-                            sp6E = (gSaveContext.cutsceneIndex & 0xF) + 4;
+                            csOffset = (gSaveContext.cutsceneIndex & 0xF) + 4;
```

Comment:
```suggestion
                            sceneSetupIndex = (gSaveContext.cutsceneIndex & 0xF) + 4;
```
since it's used in a similar fashion to `gSaveContext.sceneSetupIndex`
https://github.com/zeldaret/oot/blob/8926b0858247a1861a045af9cf33ce616054945e/src/code/z_play.c#L290

Reply:
scene setup index is used for both the age/time of day offset and cutscenes.
This is specifically the offset as designated by the cutscenes, which is why it does the + 4 to get passed the age/time stuff. So I think this warrants being named after cutscenes specifically

Reply:
nvm I understand. its the cs offset + 4 which now makes it the scene setup index